### DISCUSSION TITLE
A bug: No image file found at specified input path

### DISCUSSION THREAD

**Discussion by guzhangyu** [visioncortex/vtracer#84] [Q&A]
I run "vtracer --input test.png --output test.svg", and get the result bellow:
`
thread 'main' panicked at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/vtracer-0.6.3/src/main.rs:279:13:

Conversion failed with error message: No image file found at specified input path

stack backtrace:

   0:     0x56496133c0f5 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h1e1a1972118942ad

   1:     0x564961360ebb - core::fmt::write::hc090a2ffd6b28c4a

   2:     0x564961339bcf - std::io::Write::write_fmt::h8898bac6ff039a23

   3:     0x56496133bece - std::sys_common::backtrace::print::ha96650907276675e

   4:     0x56496133d339 - std::panicking::default_hook::{{closure}}::h215c2a0a8346e0e0

   5:     0x56496133d07d - std::panicking::default_hook::h207342be97478370

   6:     0x56496133d7d3 - std::panicking::rust_panic_with_hook::hac8bdceee1e4fe2c

   7:     0x56496133d6b4 - std::panicking::begin_panic_handler::{{closure}}::h00d785e82757ce3c

   8:     0x56496133c5b9 - std::sys_common::backtrace::__rust_end_short_backtrace::h1628d957bcd06996

   9:     0x56496133d3e7 - rust_begin_unwind

  10:     0x56496120a373 - core::panicking::panic_fmt::hdc63834ffaaefae5

  11:     0x564961211bed - vtracer::main::h5980927ca50eb156

  12:     0x56496120b3d3 - std::sys_common::backtrace::__rust_begin_short_backtrace::hfb1c7e4ff473a9c0

  13:     0x5649612123f9 - std::rt::lang_start::{{closure}}::h1c65a6cde4bb0b62

  14:     0x564961333d30 - std::rt::lang_start_internal::h3ed4fe7b2f419135

  15:     0x5649612123e5 - main

  16:     0x7f249d6ba555 - __libc_start_main

  17:     0x56496120aa20 - <unknown>

  18:                0x0 - <unknown>
`
In fact the input image exists.

**Comment by flagman** [comment]
Ensure that the image you're providing is actually a PNG file, not just a JPEG with a .png extension."
